### ISSUE TITLE
Create a native mobile app SDK & React web UI components

### ISSUE THREAD

**Issue by emaalouf** [cracker0dks/Accelerator#34]
I think this is a very important topic, we need to create a an SDK for mobile, now here is what I think works with the all mobile platforms:

- https://github.com/flutter-webrtc/flutter-webrtc-demo

**Comment by cracker0dks** [comment]
Yeah, I looked into that, but it's too much work... please consider that we have no paid dev, this is all hobby... so afaik we should make the webclient more responsive so it works better on IOS and Android if we need to support mobile.
